code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -219,7 +221,7 @@
 	spare.domutcheck()
 	spare.Move(get_step(H.loc, pick(NORTH,SOUTH,EAST,WEST)))
 
-	H.blood_volume = BLOOD_VOLUME_SAFE
+	H.blood_volume *= 0.45

```

Comment:
This seems unrelated.

Reply:
I forgot about that, i used it for testing

I'll either pr it separately or document it if you want

Reply:
Please do either.

Reply:
Documented and kept in; it's not very related but i'd rather not make a whole new pr for a likely uncontroversial change